source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,195 @@
+
+.. warning:: **DO NOT** attempt to "repair" damaged or non-functional batteries.
+
+Battery Construction & Tools
+-----------------------------
+
+Battery Leads
+^^^^^^^^^^^^^
+
+Battery leads must be copper, minimum size (cross section) 6 AWG (16mm2, 7 SWG) and maximum length 12", with red or black insulation matched to polarity, and an Anderson SB connector. Standard 6AWG copper leads with Pink/Red SB50 battery leads often come in the Kit of Parts and are sold by FRC vendors.
+
+Lead Cables
+~~~~~~~~~~~
+
+Tinned, annealed, or coated copper is allowed. Do not use CCA (copper clad aluminum), aluminum, or other non-copper base metal. The conductor metal is normally printed on the outside of the insulation with the other cable ratings.
+
+Wire size 6AWG is sufficient for almost all robots and fits standard SB50 contacts. A small number of teams adopt larger wire sizes for marginal performance benefits.
+
+Higher strand count wire (sometimes sold as "Flex" or "welding wire") has a smaller bend radius, which makes it easier to route, and a higher fatigue limit. There is no strand count requirement, but 84/25 (84 strand "flex" hookup wire) and 259/30 (259 strand "welding wire") will both be *much* easier to work with than 19/0.0372 (19 strand hookup wire).
+
+The insulation of the +12V wire must be red, and the ground must be black. There is no explicit insulation temperature rating requirement, but any blackened or damaged insulation means the wire needs to be replaced: off hand, 105C is plenty and lower will work for almost all robots. There is no insulation voltage rating requirement, lower is better for thinner insulation.
+
+SB Connector
+~~~~~~~~~~~~~
+
+The Anderson SB Connector may be the standard Pink/Red SB50, or another Anderson SB connector. Teams are *STRONGLY* recommended to use the Pink/Red SB50 for interoperability: the other colors and sizes of housings will not intermate, and you will be unable to borrow batteries or chargers.
+
+Follow manufacturer's instructions to crimp contacts and assemble the leads into Anderson SB connectors. A small flathead screwdriver can help to insert the contacts (push on the contact, not on the wire insulation), or it can help to disengage the internal latch if the contact is in the wrong slot or upside down.
+
+Battery Lugs
+^^^^^^^^^^^^^
+
+Compression lugs ("crimp lugs") for #10 bolt battery tabs (~0.2" hole diameter) are available online and through electrical supply houses, sold by the accepted wire sizes in AWG (or mm2) and post diameter ("bolt size", "hole diameter"). Higher end vendors will also distinguish between Standard (~19) and Flex (>80) strand counts in their lug catalogs. Some vendors also offer right angle lugs, in addition to more common straight styles. Follow manufacturer's instructions to crimp the lugs.
```

Comment:
We're explicitly on the battery assembly side, here, so it's very simple to keep this at 'follow instructions, this vocabulary will help you read a catalog page', and I am intentionally vague about "sold by wire size range they accept". 

I really don't want to get into "This is the WPILib Method to Run 4AWG" because any discussion of using larger gauge wire gets into Engineering Judgment at the SB50 & PDP immediately.

Reply:
Fair enough, I don't think there are many 6AWG lug that don't fit so that seems fine.